intl/msg_hash_us.h

```diff
@@ -10746,11 +10754,11 @@ MSG_HASH(
    )
 MSG_HASH(
    MSG_DISK_CLOSED,
-   "Closed"
+   "Closed virtual disc tray."
```

Comment:
Maybe "Virtual disc tray is closed." is more natural?

Reply:
Adjusting the actual phrasing is not the purpose of this PR. Not to mention, that this is a log message describing what happened, thus I think the current phrasing is fitting as is.

Reply:
This log message also appears as a notification, so I do not know.